pkgs/development/libraries/botan/generic.nix

```
@@ -1,6 +1,7 @@
 { stdenv, fetchurl, python, bzip2, zlib, gmp, openssl, boost
 # Passed by version specific builders
 , baseVersion, revision, sha256
+, srcext ? "tar.xz"
```

lets keep these camelCase: ``
```suggestion
, sourceExtension ? "tar.xz"
```